### PR TITLE
amie.so app

### LINKED ISSUE #5787: [CAL-504] amie.so app
null

<sub>[CAL-504](https://linear.app/calcom/issue/CAL-504/amieso-app)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
image

### CODE REVIEW

File: packages/app-store/amie/config.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "/*": "Don't modify slug - If required, do it using cli edit command",
+  "name": "Amie",
+  "slug": "amie",
+  "type": "amie_other",
+  "imageSrc": "/api/app-store/amie/icon.svg",
+  "logo": "/api/app-store/amie/icon.svg",
+  "url": "https://cal.com/apps/amie",
+  "variant": "other",
+  "categories": ["calendar"],
```
Comment (PeerRich) [inline packages/app-store/amie/config.json:10]: @hariombalhara i did the same as with vimcal but it doesnt show up in "calendar" categories

**Comment by PeerRich** [comment]
@hariombalhara can you help me? its not showing up in the "calendar" category and is in "other" instead:

CleanShot 2022-12-16 at 11 46 43

but on the detail page it says "calendar" 

CleanShot 2022-12-16 at 11 47 21


same as Vimcal, we dont want to show this on the onboarding

**Comment by PeerRich** [comment]
lets remove max-height from dialog to prevent scrolling there

File: packages/app-store/amie/api/add.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const handler: AppDeclarativeHandler = {
+  // Instead of passing appType and slug from here, api/integrations/[..args] should be able to derive and pass these directly to createCredential
+  appType: appConfig.type,
+  variant: appConfig.variant,
+  slug: appConfig.slug,
+  supportsMultipleInstalls: false,
+  handlerType: "add",
+  redirect: {
+    newTab: true,
+    url: "https://amie.so/signup",
```
Comment (hariombalhara) [inline packages/app-store/amie/api/add.ts:15]: Why is it a redirect to signup? Shouldn't it talk about how the integration would work

**Review by hariombalhara — APPROVED** [review]
Looks good except that redirect URL issue.

File: packages/app-store/amie/api/add.ts (hunk 2 above)
Comment (PeerRich) [reply packages/app-store/amie/api/add.ts:15]: for now to signup because they are in a waitlist. we're writing a blog post right now, will update then

**Pull request merged**